src/main/java/pl/magisterka/dominikszojda/entity/Breed.java

```diff
@@ -0,0 +1,29 @@
+package pl.magisterka.dominikszojda.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Setter
+@Getter
+@Table(name= "breed")
+public class Breed {
+    @Id
+    @Column
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+    @Column
+    private String name;
+    @Column
+    private String description;
+    @Column
+    private String group_id;
```

Comment:
zmienne w Javie zapisuje się `camelCase`. Spring sobie poradzi i przetłumaczy tą nazwę na `snake_case` w bazie danych. Jeżeli chcesz sam kontrolować nazwy kolumn to możesz użyć parametru `name` w `@Column`: `@Column(name = "moja_custom_nazwa")`.

Ponad to, tu masz relację do `DogGroup` więc lepiej użyć odpowiedniego typu by Spring wiedział co się dzieje i że są relacje. Przy robieniu takiej relacji zamiast `@Column` używamy `@JoinColumn` (używa się tak samo), oraz adnotację określające rodzaj relacji (w tym wypadku zgaduje że to bedzie `@ManyToOne` lub `@OneToOne`)

```suggestion
    @ManyToOne
    @JoinColumn(name = "group_id")
    private GroupEntity group;
```